src/dpp/guild.cpp

```diff
@@ -229,13 +229,20 @@ std::string guild_member::build_json(bool with_id) const {
 			j["communication_disabled_until"] = json::value_t::null;
 		}
 	}
-	if (!this->nickname.empty())
-		j["nick"] = this->nickname;
+	
+	if (!this->nickname.empty()) {
```

Comment:
You sure this works? You clear the nickname if it's not empty, and set it, if it's empty? accidentally negated i guess

Reply:
Oopps